core/time/timezone/tz_unix.odin

```diff
@@ -7,61 +7,65 @@ import "core:strings"
 import "core:path/filepath"
 import "core:time/datetime"
 
-local_tz_name :: proc(allocator := context.allocator) -> (name: string, success: bool) {
-	local_str, ok := os.lookup_env("TZ", allocator)
-	if !ok {
-		orig_localtime_path := "/etc/localtime"
-		path, err := os.absolute_path_from_relative(orig_localtime_path, allocator)
-		if err != nil {
-			// If we can't find /etc/localtime, fallback to UTC
-			if err == .ENOENT {
-				str, err2 := strings.clone("UTC", allocator)
-				if err2 != nil { return }
-				return str, true
+local_tz_name :: proc(check_env: bool, allocator := context.allocator) -> (name: string, success: bool) {
```

Comment:
Is there a reason this `check_env` was added, I don't see a real use case (when would you not check env?) and am not a fan of bool flags (because reading `local_tz_name(true)` is hard, what does `true` mean)